app/views/shared/fields/_corporate_name.html.erb

```diff
@@ -15,14 +15,14 @@
 
 		htmlOpen %= 'Corporate Name'.pluralize(fieldData.count)
 		concat htmlOpen.html_safe
+%>
 
-		fieldData.each do |datum|
-			corporateName = datum
-			concat "<li>#{corporateName}</li>".html_safe
-		end
-
-		concat htmlClose.html_safe
+	  <ul class="unstyled glyphicon-list glyphicon-list-tag">
+	    <% fieldData.each do |topic| %>
+	      <li><%= link_to topic, facet_uri(topic, 'subject_topic_sim') %></li>
```

Comment:
@mcritchlow corporate_name, personal_name and genre are still in general subject_topic_sim.  Lithology, Cruise, Scientific Name and Common Name are in their own.